### DISCUSSION TITLE
Vite Integration

### DISCUSSION THREAD

**Discussion by tsotimus** [fingerprintjs/fingerprintjs#1044] [Show and tell]
Hey everyone!

I've just released a vite-plugin for fingerprintjs.

Should make it easier to integrate via the `vite.config.ts` and it has some additional functionality to bypass browser/adblockers blocking the script from running.

Check it out and let me know what you think! 

[NPM](https://www.npmjs.com/package/vite-plugin-fingerprintjs)
[Repo](https://github.com/RockiRider/vite-plugin-fingerprintjs)

**Comment by Finesse** [comment]
Thank you, @RockiRider!

I think that [installing FingerprintJS directly](https://github.com/fingerprintjs/fingerprintjs?tab=readme-ov-file#getting-started) is not much more difficult than using your Vite plugin, so I wouldn't use it in my projects. But I understand that it can help some people, so why not.

Your React snippet caught my attention. Because the `window.fp` value changes when FingerprintJS finishes, the `useFingerprint` should trigger a render and start returning the new value, but the `useMemo` gives no chance for a result update (unless the component is recreated).
